index.html

```diff
@@ -0,0 +1,303 @@
+                            </div>
+                        </div>
+                        <div class="indiTweetIcons">
+                            <div class="indiTweetReply tastyPadding">
+                                <img class="indiTweetIconResizer" src="./assets/icons/messageBubbleIcon.svg" alt="reply icon">
+                            </div>
+                            <div class="indiTweetReTweet tastyPadding">
+                                <img class="indiTweetIconResizer" src="./assets/icons/repeatIcon.svg" alt="retweet icon">
+                            </div>
+                            <div class="indiTweetHeart tastyPadding">
+                                <img class="indiTweetIconResizer" src="./assets/icons/heartIcon.svg" alt="heart icon">
+                            </div>
+                            <div class="indiTweetUpload tastyPadding">
+                                <img class="indiTweetIconResizer" src="./assets/icons/uploadIcon.svg" alt="upload icon">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="individualTweets bgColor">
+                        <div class="indiTweetPFP">
+                            <img src="./assets/profilePicture/bulbasaurProfilePic.jpg" alt="profile picture">
+                        </div>
+                        <div class="indiTweetNames">
+                            <div class="indiTweetDisplayName">
+                                <span>Bantah-sawr</span>
+                            </div>
+                            <div class="indiTweetUsername">
+                                <span>@totally_real_bulbasaur</span>
+                            </div>
+                            <div class="indiTweetLittleDot">
+                                <span>-</span>
+                            </div>
+                            <div class="indiTweetTime">
+                                <span>3m</span>
+                            </div>
+                        </div>
+                        <div class="indiTweetMainText">
+                            <div class="indiTweet">
+                                <span><span class="blueText">@totally_real_charmander</span> simmer down bud... here is a detailed explanation of why youre wrong: 
+                                Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi dolorem, nostrum officiis repellendus officia maiores blanditiis, 
+                                mollitia accusamus sequi recusandae exercitationem labore sed consequatur facere similique reprehenderit unde magni voluptas. 1/19</span>
+                            </div>
+                        </div>
+                        <div class="indiTweetIcons">
+                            <div class="indiTweetReply tastyPadding">
```

Comment:
maybe put all these in anchors tags so they come up as clickable or if there's another way to do that